cloud-controlplane/x-topics/quickstart.md

```diff
@@ -50,7 +52,30 @@ The Response block should populate with an `access_token` value. Copy the string
 
 ### Serverless
 
-1. Make a Get Resource Group request to retrieve the default resource group ID. Pass this ID later when you call the Create Serverless Cluster endpoint.
-1. Make a Get Serverless Regions request to see available regions.
-1. Create a cluster by making a Create Serverless Cluster request, passing in the default resource group ID and desired cloud region in the request.
+1. In the page header, click **API Explorer**.
+1. On the **Choose an operation** dropdown, select **Create resource group**.
+1. Click **Get token**. You may be prompted to log in to the Redpanda Cloud UI. After you log in, the browser automatically redirects you back to the Create resource group endpoint in the API Explorer.
+1. Prepare your Create resource group request.
+  1. Under **Body**, click **+ Add** and provide a name for your resource group. A resource group is a container to organize your Redpanda Cloud resources, such as clusters and networks.
+  1. Click **Send request**. If successful, the response returns a resource group ID. Pass this ID later when you make a Create Serverless cluster request. 
+1. On the dropdown, select **Create Serverless cluster**. 
+1. Prepare your Create Serverless cluster request.
+  1. Make a Get Serverless Regions request to see available regions.
+  1. In the request body, use the resource group ID and desired cloud region.
+  1. Click **Send request**. Note that this endpoint returns a long-running operation. The response returns a Serverless cluster ID in `metadata.cluster_id`. To check the operation state, make a **Get operation** request with the `operation.id`.
+
+## Next steps: try the Data Plane APIs
+
+1. Retrieve your cluster's data plane API URL by making a **Get cluster** (BYOC, Dedicated) or **Get Serverless cluster** (Serverless) request in the API Explorer.
+1. Save the value of `dataplane_api.url` from the response body.
+1. From the **Redpanda APIs** selector, go to **Cloud Data Plane API**.
+1. Select an operation, for example **Create topic** or **List users**. 
+1. In the URL field, add the data plane API URL. You can now make Data Plane API requests to your target cluster.
+
+See also: [Data Plane API Quickstart](https://docs.redpanda.com/api/doc/cloud-dataplane/topic/topic-quickstart)
```

Comment:
Could we possibly use relative URLs? These technically don't live in the same spec but would something like `../../../cloud-dataplane/topic/topic-quickstart` (however far back it needs to go to get to a base URL) work? Not sure when these reference would be evaluated however.